ubcpi/static/js/spec/ubcpi_edit_spec.js

```diff
@@ -381,7 +381,76 @@ describe('UBCPI_Edit module', function () {
                 expect(mockNotify.calls.count()).toBe(3);
             });
         });
-    })
+    });
+    describe('EditSettingsController', function () {
```

Comment:
Is there not any existing test suite w.r.t editing? Writing an entire suite for an edge case is kind-of overkill.

Reply:
Yes, there is. But for our case, we need to set up the mock data differently (without **correct_rationale** value) to cover our case. We are only checking the initial state of the controller in this test suite. The existing suite has other tests as well.